Assets/Scripts/Scenes/InGame/Player/PlayerStatus.cs

```diff
@@ -0,0 +1,188 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UniRx;
+using System;
+
+/// <summary>
+/// �v���C���[�̃X�e�[�^�X���Ǘ�����N���X
+/// </summary>
+namespace Scenes.Ingame.Player
+{
+
+    public class PlayerStatus : MonoBehaviour
+    {
+        //�v���C���[�̃f�[�^�x�[�X(���u��)
+        private int _playerID = 0;
+        private int _healthBase = 100;
+        private int _staminaBase = 100;
+        private int _sanBase = 100;
+
+        //���݂̃X�e�[�^�X�̕ϐ��i����l�b�g���[�N���\��j
+        //�������ɂ��Ă�����̓f�[�^�x�[�X���Q�Ƃ��čs���悤�ɂ���B
+        [SerializeField] private IntReactiveProperty _health = new IntReactiveProperty(0);//HP.�[���ɂȂ�Ǝ��S
+        [SerializeField] private IntReactiveProperty _stamina = new IntReactiveProperty(0);//�X�^�~�i
+        [SerializeField] private IntReactiveProperty _san = new IntReactiveProperty(0);//SAN�l
```

Comment:
```suggestion
[SerializeField] private IntReactiveProperty _health = new IntReactiveProperty();//HP.�[���ɂȂ�Ǝ��S
[SerializeField] private IntReactiveProperty _stamina = new IntReactiveProperty();//�X�^�~�i
[SerializeField] private IntReactiveProperty _san = new IntReactiveProperty();//SAN�l
``` 
初期値をInitで入れる場合は、こちたで初期値を入れる必要はないと思います。